packages/loader/container-loader/src/deltaManager.ts

```diff
@@ -1063,11 +1062,6 @@ export class DeltaManager<TConnectionManager extends IConnectionManager>
 			0x267 /* "lastObservedSeqNumber should be updated first" */,
 		);
 
-		// Back-compat for older server with no term
-		if (message.term === undefined) {
-			message.term = OnlyValidTermValue;
-		}
-
```

Comment:
Does this mean that client v2int/7.0 (or whichever ends up with this change) stops being compatible with some version of server? Do we know which version that is? I think it'd be a very important bit for release notes.

Reply:
No should be fine - older versions of client that would have asserted also always had this fixup.

This fixup is unnecessary in modern versions of the client, but was maintained just to keep snapshot tests compatible until this real removal happened.

Only assert that would have been scary: https://github.com/microsoft/FluidFramework/pull/14959#discussion_r1158902855

More commentary on why this fixup is still here (re: snapshot tests): https://github.com/microsoft/FluidFramework/pull/15166#issue-1673846587